### ISSUE TITLE
Failed to precompile Pluto

### ISSUE THREAD

**Issue by NAThompson** [fonsp/Pluto.jl#3002]
Apologies for the lack of a video, but I believe this one is simple enough that it's unwarranted:

```julia
~ ❯❯❯ julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.12.0-DEV.854 (2024-07-14)
 _/ |\__'_|_|_|\__'_|  |  Commit b88f64f16c (32 days old master)
|__/                   |

(@v1.12) pkg> add Pluto
Precompiling all packages...
  ✗ Pluto
  27 dependencies successfully precompiled in 11 seconds. 161 already precompiled.
  1 dependency had output during precompilation:
┌ FuzzyCompletions
│  WARNING: could not import REPLCompletions.afterusing into FuzzyCompletions
└
  1 dependency errored.
  For a report of the errors see `julia> err`. To retry use `pkg> precompile`

(@v1.12) pkg> precompile
Precompiling all packages...
  ✗ Pluto
  0 dependencies successfully precompiled in 4 seconds. 188 already precompiled.

ERROR: The following 1 direct dependency failed to precompile:

Pluto

Failed to precompile Pluto [c3e4b0f8-55cb-11ea-2926-15256bba5781] to "~/.julia/compiled/v1.12/Pluto/jl_k1j7oF".
ERROR: LoadError: UndefVarError: `complete_symbol` not defined in `REPL.REPLCompletions`
Suggestion: check for spelling errors or missing imports.
Stacktrace:
  [1] getproperty
    @ ./Base.jl:42 [inlined]
  [2] complete_keyword_argument(partial::String, last_idx::Int64, context_module::Module)
    @ FuzzyCompletions ~/.julia/packages/FuzzyCompletions/SX6dK/src/FuzzyCompletions.jl:464
  [3] completions(string::String, pos::Int64, context_module::Module, shift::Bool; enable_questionmark_methods::Bool, enable_dict_keys::Bool, enable_expanduser::Bool, enable_path::Bool, enable_bslash::Bool, enable_methods::Bool, enable_keyword_argument::Bool, enable_packages::Bool)
    @ FuzzyCompletions ~/.julia/packages/FuzzyCompletions/SX6dK/src/FuzzyCompletions.jl:730
  [4] completions (repeats 2 times)
    @ ~/.julia/packages/FuzzyCompletions/SX6dK/src/FuzzyCompletions.jl:604 [inlined]
  [5] completion_fetcher(query::String, pos::Int64, workspace::Module)
    @ Pluto.PlutoRunner ~/.julia/packages/Pluto/RveGH/src/runner/PlutoRunner/src/PlutoRunner.jl:2036
  [6] #129
    @ ~/.julia/packages/Pluto/RveGH/src/runner/PlutoRunner/src/precompile.jl:21 [inlined]
  [7] (::Base.var"#68#69"{Pluto.PlutoRunner.var"#129#130"})(::Nothing, x::String)
    @ Base ./tuple.jl:693
  [8] BottomRF
    @ ./reduce.jl:78 [inlined]
  [9] afoldl
    @ ./operators.jl:597 [inlined]
 [10] _foldl_impl
    @ ./reduce.jl:60 [inlined]
 [11] foldl_impl
    @ ./reduce.jl:40 [inlined]
 [12] mapfoldl_impl
    @ ./reduce.jl:36 [inlined]
 [13] mapfoldl
    @ ./reduce.jl:167 [inlined]
 [14] foldl
    @ ./reduce.jl:190 [inlined]
 [15] foreach(f::Pluto.PlutoRunner.var"#129#130", itr::NTuple{5, String})
    @ Base ./tuple.jl:693
 [16] macro expansion
    @ ~/.julia/packages/Pluto/RveGH/src/runner/PlutoRunner/src/precompile.jl:20 [inlined]
 [17] macro expansion
    @ ~/.julia/packages/PrecompileTools/L8A3n/src/workloads.jl:78 [inlined]
 [18] top-level scope
    @ ~/.julia/packages/Pluto/RveGH/src/runner/PlutoRunner/src/precompile.jl:102
 [19] include(mod::Module, _path::String)
    @ Base ./Base.jl:580
 [20] include(x::String)
    @ Pluto.PlutoRunner ~/.julia/packages/Pluto/RveGH/src/runner/PlutoRunner/src/PlutoRunner.jl:24
 [21] top-level scope
    @ ~/.julia/packages/Pluto/RveGH/src/runner/PlutoRunner/src/PlutoRunner.jl:2874
 [22] include(mod::Module, _path::String)
    @ Base ./Base.jl:580
 [23] include(x::String)
    @ Pluto ~/.julia/packages/Pluto/RveGH/src/Pluto.jl:11
 [24] top-level scope
    @ ~/.julia/packages/Pluto/RveGH/src/Pluto.jl:52
 [25] include
    @ ./Base.jl:580 [inlined]
 [26] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::Nothing)
    @ Base ./loading.jl:2797
 [27] top-level scope
    @ stdin:4
 [28] eval
    @ ./boot.jl:438 [inlined]
 [29] include_string(mapexpr::typeof(identity), mod::Module, code::String, filename::String)
    @ Base ./loading.jl:2619
 [30] include_string
    @ ./loading.jl:2629 [inlined]
 [31] exec_options(opts::Base.JLOptions)
    @ Base ./client.jl:322
 [32] _start()
    @ Base ./client.jl:532
in expression starting at ~/.julia/packages/Pluto/RveGH/src/runner/PlutoRunner/src/precompile.jl:6
in expression starting at ~/.julia/packages/Pluto/RveGH/src/runner/PlutoRunner/src/PlutoRunner.jl:24
in expression starting at ~/.julia/packages/Pluto/RveGH/src/Pluto.jl:1
in expression starting at stdin:
```

**Comment by fonsp** [comment]
Hey! This is because of https://github.com/fonsp/Pluto.jl/issues/2939, which should have been fixed by FuzzyCompletions 0.5.5.

Could you do two things:
- Make sure that FuzzyCompletions 0.5.5 is installed, and try again on Julia 1.12 (nightly). If it still does not work, send a copy of `pkg> status -m`.
- Try again on Julia 1.11 or Julia 1.6 and report if it fixed the problem.

Thanks!

**Comment by fonsp** [comment]
No worries @NAThompson ! Could you still answer those two questions if you have time?
